apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/LoadRdaJsonApp.java

```diff
@@ -44,8 +44,16 @@
  * data so either or both can be provided as needed.
  */
 public class LoadRdaJsonApp {
+  /** Logger for RDA Json App. */
   private static final Logger logger = LoggerFactory.getLogger(LoadRdaJsonApp.class);
```

Comment:
since its a static logger, just change the name to LOGGER and you won't need to document it. (Special logger exception)

Reply:
done